app/pages/reservation/reservation-information/_reservation-information.scss

```diff
@@ -18,3 +18,67 @@
     margin-bottom: 10px;
   }
 }
+
+.app-InternalReservationForm__checkbox {
+  display: block;
+  position: relative;
+  padding-left: 35px;
+  padding-bottom: 12px;
+  margin-bottom: 12px;
+  cursor: pointer;
+  font-size: 22px;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+  input {
+    position: absolute;
+    opacity: 0;
+    cursor: pointer;
+    height: 0;
+    width: 0;
+    &:checked {
+      &~.custom-checkmark {
+        background-color: $white;
+        border: 2px solid $blue;
+        &:after {
+          display: block;
+        }
+      }
+    }
+  }
+  &:hover {
+    input {
+      &~.custom-checkmark {
+        background-color: $white;
+      }
+    }
+  }
+  .custom-checkmark {
+    &:after {
+      left: 5px;
+      top: 1px;
+      width: 5px;
+      height: 10px;
+      border: solid $black;
+      border-width: 0 3px 3px 0;
+      -webkit-transform: rotate(45deg);
+      -ms-transform: rotate(45deg);
+      transform: rotate(45deg);
+    }
+  }
+}
+.custom-checkmark {
```

Comment:
this wont be needed if you style the original checkbox instead of creating new decorated checkbox

Reply:
This hacky css is removed.